automation/test.sh

```diff
@@ -50,51 +50,73 @@ kubectl() { cluster/kubectl.sh "$@"; }
 export NAMESPACE="${NAMESPACE:-kube-system}"
 
 # Make sure that the VM is properly shut down on exit
-trap '{ make cluster-down; }' EXIT
+trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGKILL SIGSTOP
```

Comment:
Can you remove `SIGKILL`? It can't be trapped.

Reply:
True, thx